examples/NeuralNetwork-load/sketch.js

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2024 ml5
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+// This example will load a pre-trained model that recognizes hand gestures for rock paper scissors.
+
+let classifier;
+let handPose;
+let video;
+let hands = [];
+let classification = "";
+let isModelLoaded = false;
+
+function preload() {
+  // Load the handPose model
+  handPose = ml5.handPose();
+}
+
+function setup() {
+  createCanvas(640, 480);
+
+  // Create the webcam video and hide it
+  video = createCapture(VIDEO);
+  video.size(width, height);
+  video.hide();
+
+  // For this example to work across all browsers
+  // "webgl" or "cpu" needs to be set as the backend
+  ml5.setBackend("webgl");
```

Comment:
Is this needed or can it just work with the default?

Reply:
The example works without it! However, I do remember discussing this and it is there for some reason, though I don't remember what it is exactly.